src/commands/PublishCommand.js

```diff
@@ -542,6 +542,8 @@ export default class PublishCommand extends Command {
 
     this.updates.forEach((update) => {
       this.execScript(update.package, "prepublish");
+      this.execScript(update.package, "prepare");
```

Comment:
this on the other hand could be parallelized to speed up the process, Im not sure what to use here though, I guess it would need to be a combo of `PackageUtilities.runParallelBatches` and `async.series`. I see the desired flow as:
- parallel execution of `async.series` per each package
- on completion one more parallel batch of publish tasks

I think its the safest one